File: Application/Services/Operations/Outsorced/Dtos/BillingFromDto.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using Application.Services.Operations.Main.Customers.Dtos;
using Application.Services.Operations.Main.Partners.Dtos;
using Domain.Entities;

namespace Application.Services.Operations.Outsourced.Dtos
{
    public class BillingFromDto
    {
        public int Id { get; set; }

        public int? PartnerId { get; set; }
        public PartnerDto Partner { get; set; }

        public int? CustomerId { get; set; }
        public CustomerDto Customer { get; set; }

        public bool Base { get; set; }
    }
}

File: Views/src/components/out-sourced/collect-deliver/components/dto/billing-from-dto.ts
import { CustomerDto } from "src/components/main/customer/dtos/customer-dto";
import { PartnerDto } from "src/components/main/partner/dto/partner-dto";


export class BillingFromDto {
  id: number;

  partnerId: number;
  partner: PartnerDto;

  customerId: number;
  customer: CustomerDto;

  base: boolean;
}
